vm/src/stdlib/os.rs

```diff
@@ -1195,6 +1197,50 @@ mod _os {
         }
     }
 
+    #[derive(FromArgs)]
+    struct CopyFileRangeArgs {
```

Comment:
Could you put the `cfg()` on here too? So there aren't `unused` warnings on other platforms?

Reply:
Maybe `UtimeArgs` need to move behind `cfg()` also?
```rust
#[derive(FromArgs)]
    struct UtimeArgs {
    }
```

Reply:
Oh, sure, go for it if you want